### DISCUSSION TITLE
Mirror to GitLab

### DISCUSSION THREAD

**Discussion by fakeid30** [aapatre/Automatic-Udemy-Course-Enroller-GET-PAID-UDEMY-COURSES-for-FREE#154] [General]
Let's discuss how we are going to mirror this code to GitLab.

**Comment by fakeid30** [comment]
Basic mirroring done : https://gitlab.com/the-automators/Automatic-Udemy-Course-Enroller-GET-PAID-UDEMY-COURSES-for-FREE/

Need to fix CI/CD on GitLab. @cullzie can you take a look at it please? Seems like they are not compatible with GitHub's syntax.

**Reply by cullzie** [reply]
Do we want to maintain CI on both though? If its working on github it should be good enough. If we want to migrate to using gitlab we can disable it here. Just don't want to have duplicated effort

**Reply by fakeid30** [reply]
GitLab offers better CI/CD than GitHub. I think we stay here for the main code but we go to GitLab for CI/CD. I'll take a look into porting the GH yml to GL later this week. 

Given the amount of recognition we are getting I don't think we are moving to GL anytime soon.

**Reply by cullzie** [reply]
Ye that makes sense. Its not a priority at the moment but would be nice to get up and running

**Reply by fakeid30** [reply]
Really wish GL's UI was as polished as GH

**Comment by fakeid30** [comment]
@cullzie https://github.com/aapatre/Automatic-Udemy-Course-Enroller-GET-PAID-UDEMY-COURSES-for-FREE/actions/runs/425532826 

CI is breaking

**Reply by cullzie** [reply]
Account we are using is not logging in correctly. Checking that we're logged in is timing out. 
I can't see the github secrets for username/password. Might just need to create a new user/password and put it in there

**Reply by fakeid30** [reply]
do you have any test account ?

**Reply by fakeid30** [reply]
if so , send me an email , i will have to ask @aapatre to change it

**Reply by fakeid30** [reply]
@cullzie the logging in works when CI is invoked on master, just not the development

**Reply by cullzie** [reply]
No I just create them with 10 minute mail. Don't have any saved anywhere sorry

**Reply by fakeid30** [reply]
ok i asked antariksh to change the email and pass, lets see if it works

**Reply by cullzie** [reply]
> @cullzie the logging in works when CI is invoked on master, just not the development

Its not really working on master. The enroll is timing out the past while based on the logs

**Reply by fakeid30** [reply]
> Its not really working on master. The enroll is timing out the past while based on the logs

New secrets have been added. But still failing
